commitlint/testcommitlint.sh

```diff
@@ -451,5 +451,11 @@ Changelog: none
 Signed-off-by: Kristian Amlie <kristian.amlie@northern.tech>"
 
 
+assert "true" \
+       "Accept standard git-revert commits" \
+"Revert build(docker-client): Add support for building the cpp-client
```

Comment:
In reality, the title of the old commit is quoted: `Revert "build(docker-client): Add support for building the cpp-client"`. Will this still work with the parser?

Reply:
hah, good catch! I copy pasted, and was like: Wtf is these extra quotes doing here...

Reply:
And it broke... :disappointed:

Reply:
Fixed.

One other thing, is that we probably shouldn't parse these commits at all.

Reply:
> One other thing, is that we probably shouldn't parse these commits at all.

We have to parse the commits. The `git revert` will only keep the title and "This reverts commit blablabla", but the user most likely appends some context on why the revert is there and possibly have a changelog.